config/config.go

```diff
@@ -70,6 +71,10 @@ type Config struct {
 func (c Config) Validate() bool {
 	errs := []error{}
 
+	if runtime.GOOS == "windows" && c.SSHExternal {
+		errs = append(errs, fmt.Errorf("-ssh-external cannot be used on windows."))
```

Comment:
Amended the change and added gometalinter checks to my editor to avoid these issues in the future.